core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/RenderingTests.kt

```diff
@@ -79,25 +84,31 @@ class RenderingTests : JupyterReplTestCase() {
 
     @Test
     fun `test kotlin notebook plugin utils rows subset`() {
-        @Language("kts")
-        val result = exec<MimeTypedResult>(
+        val json = executeAndParse(
             """
             data class Row(val id: Int)
             val df = (1..100).map { Row(it) }.toDataFrame()
             KotlinNotebookPluginUtils.getRowsSubsetForRendering(df, 20 , 50)
             """.trimIndent()
         )
 
-        val json = parseDataframeJson(result)
-
-        json.int("nrow") shouldBe 30
-        json.int("ncol") shouldBe 1
+        assertDataFrameDimensions(json, 30, 1)
 
         val rows = json.array<JsonArray<*>>("kotlin_dataframe")!!
         rows.getObj(0).int("id") shouldBe 21
         rows.getObj(rows.lastIndex).int("id") shouldBe 50
     }
 
+    private fun executeAndParse(script: String): JsonObject {
```

Comment:
some small kdoc for these might be nice :) otherwise, it cleans up repeated statements nicely